src/plugins/data/common/expressions/expression_types/kibana_datatable.ts

```diff
@@ -21,9 +21,15 @@ import { map } from 'lodash';
 
 const name = 'kibana_datatable';
 
+export interface SerializedFieldFormat<TParams = object> {
```

Comment:
That's how I started out over here (https://github.com/elastic/kibana/pull/38874/commits/554857954addb35eb6a78762134abf3f9bb1343e), but somehow the type check fails in completely unrelated places if I do that - something about importing legacy stuff from the new world. Then I thought it's maybe even cleaner that way because the types of the data table are all defined in one place.

Maybe this was a configuration error already fixed, I'll move it over and check.

Reply:
Ah I didn't think about that @flash1293 -- you are totally right. You can import new platform into legacy, but you cannot import legacy into new platform.

Not sure what makes the most then, since we definitely won't want to keep this in with the datatable type longer term. Maybe instead we should stick it in the `types/common.ts` file for now? I just don't want it to get lost in `expression_types/kibana_datatable`

Reply:
Moved it over to common